torch/_subclasses/fake_tensor.py

```diff
@@ -802,6 +802,45 @@ class DispatchCacheInfo:
     size: int
 
 
+cached_fake_script_obj = {}
+
+
+def _fakify_script_object(x):
+    if hash(x) in cached_fake_script_obj:
+        return cached_fake_script_obj[hash(x)]
```

Comment:
Why do we need the caching? Do you have a concrete example?

Reply:
tq should already be a FakeTensorQueue when tracing -- Line 1 and Line 2 should not need to create new FakeTensorQueue. What am I missing here?